app/scripts/metamask-controller.js

```diff
@@ -2882,6 +3165,35 @@ export default class MetamaskController extends EventEmitter {
         ),
       }),
     );
+
+    ///: BEGIN:ONLY_INCLUDE_IN(flask)
+    engine.push(
+      createSnapMethodMiddleware(subjectType === SUBJECT_TYPES.SNAP, {
+        getAppKey: this.getAppKeyForDomain.bind(this, origin),
+        getSnaps: this.snapController.getPermittedSnaps.bind(
+          this.snapController,
+          origin,
+        ),
+        // TODO:flask Fix in skunkworks (rpc-methods)
```

Comment:
Same question here: fix what?

Reply:
I went through the commit history and can't find this comment anywhere. I recall fixing it, but not what the problem was. Whatever the case, it's not there anymore :man_shrugging: